### PR TITLE
Improve default_power_level_content_override documentation

### PR DESCRIPTION
**Pull request by benbz**
### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://element-hq.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
  - Feel free to credit yourself, by adding a sentence "Contributed by @github_username." or "Contributed by [Your Name]." to the end of the entry.
* [x] [Code style](https://element-hq.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://element-hq.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

The existing docs make it clear that setting `default_power_level_content_override` for a preset removes all the defaults for that preset. The docs don't say what the defaults are which makes it hard to keep the defaults too

### CODE REVIEW

File: changelog.d/17451.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+Improve documentation on default_power_level_content_override.
```
Comment (anoadragon453) [inline changelog.d/17451.doc:1]:
```suggestion
Improve documentation on `default_power_level_content_override`.
```

**Comment by benbz** [comment]
> Thanks for this - I can see how this documentation in its current form would be pretty confusing. I think ideally we'd recursively merge in new option so that the defaults would not need to be specified. But this is helpful in the meantime.

100% agree that a recursive merge would be better but that's probably beyond what I can help with right now

File: changelog.d/17451.doc [hunk 2]
```diff
@@ -0,0 +1 @@
+Improve documentation on `default_power_level_content_override`.
```
Comment (anoadragon453) [inline changelog.d/17451.doc:1]:
```suggestion
Improve documentation for the [`default_power_level_content_override`](https://element-hq.github.io/synapse/latest/usage/configuration/config_documentation.html#default_power_level_content_override) config option.
```

**Review by anoadragon453 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**